app/views/layouts/application.html.erb

```diff
@@ -10,6 +11,14 @@
   </head>
 
   <body>
-    <%= yield %>
+    <%= render 'layouts/headeer' %>
+
+    <div class="container pt-4 mb-0">
+      <%=render 'shared/error_message' %>
```

Comment:
You don't have a partial file for **Error_message** yet so please delete this line.